wlauto/workloads/geekbench/__init__.py

```diff
@@ -101,7 +103,8 @@ def __init__(self, device, **kwargs):
         super(Geekbench, self).__init__(device, **kwargs)
         self.uiauto_params['version'] = self.version
         self.uiauto_params['times'] = self.times
-        self.run_timeout = 10 * 60 * self.times
+        if self.times > 1:
+            self.run_timeout = 15 * 60 * self.times
```

Comment:
This silently overwrites user-specified `run_timeout` when `times` is greater than one. Ether

a) replace this with `self.run_timeout = self.run_timeout * self.times` and make it clear in the documentation for the `run_timeout` parameter that it will be multiplied by the number of times, or
b) respect the user-provided timeout regardless of the the value of `times`.